packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -280,6 +286,11 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
         return false;
       }
     }
+    if (_initialPointer != null &&
```

Comment:
Don't even think we need the first check. Since event.pointer is not nullable.

Reply:
I think this check right, _initialButtons follows the same construct. Both only set after this in addAllowedPointer, rather than addPointer. :)